beets/dbcore/query.py

```diff
@@ -549,20 +550,45 @@ def __init__(self, date, precision):
     def parse(cls, string):
         """Parse a date and return a `Period` object or `None` if the
         string is empty.
+        Depending on the string, the date can be absolute or
+        relative.
+        An absolute date has to be like one of the date_formats '%Y' or '%Y-%m'
+        or '%Y-%m-%d'
+        A relative date begins by '@ 'and has to follow the pattern_dq format
+        '@([+|-]?)(\d+)([y|m|w|d])'
+        - '@' indicates it's a date relative to now()
+        - the optional '+' or '-' sign, which defaults to '+' will increment or
+        decrement now() by a certain quantity
+        - that quantity can be expressed in days, weeks, months or years
+        respectively 'd', 'w', 'm', 'y'
+        Please note that this relative calculation is rather approximate as it
+        makes the assumption of 30 days per month and 365 days per year
```

Comment:
These assumptions should probably be mentioned in the "human readable" docs (query.rst) as well. Most endusers will not read in-code documentation for details on the implementation.